package.json

```diff
@@ -28,7 +28,7 @@
     "ember-sortable": "^1.12.9",
     "ember-truth-helpers": "^2.1.0",
     "tether": "^1.4.7",
-    "upfluence-oss-components": "^2.x"
+    "upfluence-oss-components": "^2.0.31"
```

Comment:
Stay on "^2.x", no?
You can do that with this command `yarn upgrade upfluence-oss-components@^2.x` and it gets last version on master automatically

Reply:
I'm dropping that change then.
`yarn.lock` is in the gitignore for some reason, which is weird AF.

Reply:
My bad you right, for this project we don't push `yarn.lock`